neo4j-admin: add page

martinohanlon opened the pull request:
<!--
Thank you for contributing!
Please fill in the following checklist, removing items that do not apply.
See also https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md
-->

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known): Neo4j 5.20**

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/tldr-pages/tldr?pullRequest=12906) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/tldr-pages/tldr?pullRequest=12906) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/tldr-pages/tldr?pullRequest=12906) it.</sub>

vitorhcl commented on pages/common/neo4j-admin.md:
```diff
@@ -0,0 +1,32 @@
+# neo4j-admin
+
+> Manage and administer a Neo4j DBMS.
+> More information: <https://neo4j.com/docs/operations-manual/current/tools/neo4j-admin/>.
```
```suggestion
> See also: `mysql`, `sqllite3`, `psql`, `usql`.
> More information: <https://neo4j.com/docs/operations-manual/current/tools/neo4j-admin/>.
```
Non-blocking (doesn't prevent merging) change: we support page suggestions, it would be nice to recommend similar tools, who looks up for DBMS commands can be interested in other similar tools.

martinohanlon replied:
Im hoping to add additional neo4j commands, so I'll add a suitable "See also:" at that time if that suits.

vitorhcl replied:
Awesome! They will be a better "See also" indeed, we can leave just one different DBMS for connecting with other DBMSes.

vitorhcl reviewed (commented):
Thank you for your contribution, your page is good in general!

LGTM after my suggestions.

spageektti reviewed (approved):
LGTM after the suggested changes. Thanks for your contribution and welcome to tldr pages!

martinohanlon commented on pages/common/neo4j-admin.md:
```diff
@@ -0,0 +1,32 @@
+
+- Load a database from a specified archive file through `stdin`:
+
+`cat {{path/to/filename.dump}} | neo4j-admin database load --from-stdin {{database_name}} --overwrite-destination=true`
```
Thanks. Tbh I followed the existing documentation 😬 .

martinohanlon commented on pages/common/neo4j-admin.md:
```diff
@@ -0,0 +1,32 @@
+# neo4j-admin
+
+> Manage and administer a Neo4j DBMS.
```
Agreed

spageektti commented on pages/common/neo4j-admin.md:
```diff
@@ -0,0 +1,32 @@
+
+- Create an archive (dump) of an offline database to a file named `database_name.dump`:
+
+`neo4j-admin database dump --to-path={{path/to/dumps}} {{database_name}}`
```
What does `path/to/dumps` mean?
If you mean a folder which you probably mean according to [this](https://neo4j.com/docs/operations-manual/current/kubernetes/operations/dump-load/#kubernetes-neo4j-load) and [this](https://neo4j.com/docs/operations-manual/current/backup-restore/restore-dump/) wouldn't be better to use `path/to/folder`?
Also in all the examples in the docs I have seen the path isn't relative.
```suggestion
`neo4j-admin database dump --to-path={{/path/to/folder}} {{database_name}}`
```

martinohanlon commented on pages/common/neo4j-admin.md:
```diff
@@ -0,0 +1,32 @@
+
+`neo4j-admin server stop`
+
+- Set the initial password of the default `neo4j` user (required before the first DBMS start):
```
Ive changed it to scan better.

martinohanlon commented:
> LGTM after the suggested changes. Thanks for your contribution and welcome to tldr pages!

Thank you for your comments. Super helpful.

I have made a further commit addressing your comments.

vitorhcl reviewed (approved):
LGTM, thank you for your contribution and for addressing our suggestions!

If you want to do more contributions to the project, beyond, as you said, the `neo4j` you are interested in, there are some issues that are [good for begginers](https://github.com/tldr-pages/tldr/labels/good%20first%20issue).

After all, feel free to contribute to the project however you want if you have such desire.

CleanMachine1 reviewed (approved):
Thanks for your contribution.

CleanMachine1 commented on pages/common/neo4j-admin.md:
```diff
@@ -0,0 +1,32 @@
+
+- Create an archive (dump) of an offline database to a file named `database_name.dump`:
+
+`neo4j-admin database dump --to-path={{/path/to/folder}} {{database_name}}`
+
+- Load a database from an archive named `database_name.dump`:
+
+`neo4j-admin database load --from-path={{/path/to/folder}} {{database_name}} --overwrite-destination=true`
```
```suggestion
`neo4j-admin database dump --to-path={{path/to/directory}} {{database_name}}`

- Load a database from an archive named `database_name.dump`:

`neo4j-admin database load --from-path={{path/to/directory}} {{database_name}} --overwrite-destination=true`
```


I think this conflicts with another review but we should use the word directory to keep in line with other pages.

Also I am unfamiliar with this command, can we omit the / here

spageektti replied:
I agree with you that this should be a `directory` instead of a `folder` that's what I meant in the previous suggestion. I never used this tool but in the docs I have seen, all the paths were prefixed with `/`, so this is why I suggested adding this.

kbdharun reviewed (approved):
Welcome to tldr and thanks for your contribution. LGTM, after the above suggestion.
